src/react_native/core.cljs

```diff
@@ -135,25 +135,27 @@
 (def use-context react/useContext)
 
 (defn use-effect
-  ([effect-fn]
-   (use-effect effect-fn []))
-  ([effect-fn deps]
+  ([handler]
+   (use-effect handler []))
```

Comment:
usually `useEffect` without deps would call the handler on every render. I wonder if it would be confusing to see it used the same way, but being called only on mount here.

Reply:
thank @clauxx very good point, so probably we should keep useEffect without changes and having use-mount and use-unmount, because in that case `(use-effect handler [])` won't be needed cc @J-Son89

Reply:
so we'll have
```clojure
(defn use-effect
  [handler deps]
   (react/useEffect
    #(let [ret (handler)]
       (if (fn? ret) ret js/undefined))
    (bean/->js deps)))

(defn use-mount
  [handler]
  (use-effect handler []))

(defn use-unmount
  [handler]
  (use-mount (fn [] handler) []))
```

Reply:
or `(react/useEffect (fn []) nil)` won't work as expected?

Reply:
> or (react/useEffect (fn []) nil) won't work as expected?

hmm, may be better to pass js/undefined explicitly

Reply:
> shouldn't it be with use-effect?

could you elaborate?

Reply:
oh right thanks!

Reply:
ok i make changes in the doc then, thank you!

Reply:
thanks @clauxx made changes, pls check one more time